### DISCUSSION TITLE
How can i limit the height of source window and simultaneously having scrolling capability?

### DISCUSSION THREAD

**Discussion by LinArcX** [hrsh7th/nvim-cmp#634] [Q&A] [answered]
For sources window, we have this option:
`sources[number].max_item_count (type: number)`

But i'm looking for an approach that limit the height of sources windows and let me scroll it simultaneously.

(We have this option in documentation window: `documentation.maxheight (type: number)`)

**Comment by hrsh7th** [comment, marked as answer]
I think `pumheight` is what you want.

**Reply by LinArcX** [reply]
Thank you so much.
